Source/RapyutaSimulationPlugins/Private/Core/RRUObjectUtils.cpp

```diff
@@ -495,21 +519,36 @@ bool URRUObjectUtils::SetMeshActorColor(AActor* InMeshActor, const FLinearColor&
     }
     else
     {
-        UE_LOG_WITH_INFO(LogRapyutaCore,
-                         Error,
-                         TEXT("SetMeshActorColor() [%s] is not ARRMeshActor or AStaticMeshActor"),
-                         *InMeshActor->GetName());
+        meshComp = Cast<UMeshComponent>(InMeshActor->GetComponentByClass(UMeshComponent::StaticClass()));
+    }
+
+    if (nullptr == meshComp)
+    {
+        UE_LOG_WITH_INFO(LogRapyutaCore, Error, TEXT("SetMeshActorColor() [%s] has NO Mesh component"), *InMeshActor->GetName());
```

Comment:
The log already prints func name
```suggestion
        UE_LOG_WITH_INFO(LogRapyutaCore, Error, TEXT("[%s] has NO Mesh component"), *InMeshActor->GetName());
```